docs/compiler/reference/data-types.md

```diff
@@ -6,21 +6,21 @@ toc_max_heading_level: 3
 
 # Data types
 
-## Cadl
+## TypeSpec
 
 ### `object` {#object}
 
 Represent any structured model.(With properties)
 
-```cadl
-model Cadl.object
+```typespec
```

Comment:
Are code fences usually the full name, the abbreviated name, or both? ```tsp might be nice as an option anyway.

Reply:
Nick raised same above for abbrev. name. I like have ability for both. If both, this would be valid. Let's revisit this post rename since we have clear token for finding all of them.

Reply:
pending pr.